scheduler.Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM ubuntu:22.04
+
+COPY ./target/release/ballista-scheduler .
+
+EXPOSE 50050
+
+CMD ./ballista-scheduler
```

Comment:
What is the goal of these dockerfiles versus the existing ones in `dev/docker`? These are for faster dev testing as opposed to building releases?

We need to start the web ui as well, which we have with the existing dockerfiles under `dev/docker`

Reply:
> What is the goal of these dockerfiles

Oh, as stated in the issue, it wasn't discoverable due to 

```find . -name *.Dockerfile```

Returning no results. I see now that it returns `./dev/docker/ballista-arm64.Dockerfile` because it is capitalised according to [the standard](https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles).

I'll amend this PR to use those instead, and capitalize them as most devops engineers would expect.